polaris-react/src/components/TruncateText/TruncateText.tsx

```diff
@@ -0,0 +1,46 @@
+import React, {useRef, useState} from 'react';
```

Comment:
Moving this component to the top-level since multiple components are now using it.